### ISSUE TITLE
[BUG] Icon color cannot be detected while using Accessibility insight for Windows

### ISSUE THREAD

**Issue by Leslie1Yang** [microsoft/accessibility-insights-windows#1387]
**Describe the bug**
 Icon color cannot be detected while using Accessibility insight for Windows

**To Reproduce**
1. Launch Accessibility insight for Windows
2. Hover the cursor to the Windows Start Menu icon or other tray icons on the taskbar
3. Accessibility insight for Windows shows "Unable to detect colors!"

**Expected behavior**
Accessibility insight for Windows should be able to detect the icon color and show the result for the color contrast

**Actual behavior**
Accessibility insight for Windows shows "Unable to detect colors!"

**Screenshots or .GIF**
please see the attachments

**Laptop:**
 - OS: [Windows 10 20H2 19042.1586] (Get the version by running `winver` from the command line)
 - Accessibility Insights for Windows Version: 1.1.1997.1
 - Target Application: Microsoft icons

**Additional context**
Priority requested - High 
need to know the color contrast guideline for the icons
![tray icon](https://user-images.githubusercontent.com/80501344/177666378-3bdec83d-e96d-4b5e-9a30-13aeb4a16aae.jpg)
![icon on the edge](https://user-images.githubusercontent.com/80501344/177666382-052bea90-cd99-4093-a1dc-3e720a1f5225.jpg)
![start menu](https://user-images.githubusercontent.com/80501344/177666383-da2f4123-2714-4a25-bcc4-546f0e8d5dc9.jpg)

**Comment by dbjorge** [comment]
Hi @Leslie1Yang,

This is expected behavior; because auto detect uses a heuristic intended specifically for text, the contrast ratio of UI components and graphical elements must be tested manually. [The "Manually check contrast ratio" section of our Color contrast documentation](https://accessibilityinsights.io/docs/windows/getstarted/colorcontrast/#manually-check-contrast-ratio) has instructions that explain how to do this.

Does that answer your question and get you unblocked?

**Comment by Leslie1Yang** [comment]
Hi Dan,
Yes, thanks for the help and I close the bug in the system.


Br,
Leslie
